src/playlist-maker.js

```diff
@@ -711,7 +735,7 @@ export default function factory(player, initialList, initialIndex = 0) {
 
   // If an initial list was given, populate the playlist with it.
   if (Array.isArray(initialList)) {
-    playlist(initialList.slice(), initialIndex);
+    playlist(initialList, initialIndex);
```

Comment:
This `slice()` was redundant. How many times do we need to shallow-clone the array?!